kramer/src/main/kotlin/kramer/MavenRepoErrors.kt

```diff
@@ -0,0 +1,44 @@
+package kramer
+
+import com.squareup.tools.maven.resolution.ArtifactResolver
+import com.squareup.tools.maven.resolution.MavenVersion
+import kramer.MavenRepo.IndexEntry
+
+internal fun MavenRepo.handleDuplicateArtifacts(repoConfig: RepoConfig) {
```

Comment:
Since we own the MavenRepo, why are these extensions? The code organization aspect of keeping them in an error file doesn't really make up for the single use extension part.

Reply:
Because then I can use its properties instead of having to pass them as functional parameters. Having them here gets them out of the way, so it increases legibility, but they ARE single use. There's no reason to use them in any other context. It's breaking up responsibilities a bit, that's all.